src/main/res/values/strings.xml

```diff
@@ -647,6 +647,10 @@
     <string name="accept_invalid_certificates">Accept invalid certificates</string>
     <string name="switch_account">Switch Profile</string>
     <string name="add_account">Add Profile</string>
+    <string name="profile_tag">Profile Tag</string>
+    <string name="enter_tag">Enter Tag</string>
+    <string name="set_tag">Set Tag</string>
```

Comment:
"set" seems redundant here, usually, we do not use that verb. the menu entry would better just read "Profile Tag" (or whatever we chose here)

Reply:
I took inspiration from your branch, kept "Tag" instead of label, but if you prefer Label we can change it

Reply:
tag is fine. and i also think "private" is an improvement over my "personal"